Add vast2nix (#1)
tenzir/awesome-vast: readme.md

## Community Projects

> - [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) A Nix-based DevOps framework for the definition of declarative and reproducible environments jobs and. 

Reviewer: ```suggestion
- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

  - check all available tasks:
```bash

> git clone https://github.com/GTrunSec/vast2nix && cd vast2nix

Reviewer: I think the getting started guide should be part of the README of your project.

>   - check all available tasks:
> ```bash
> git clone https://github.com/GTrunSec/vast2nix && cd vast2nix
> nix develop --command std list
> ```
>     

Reviewer: ```suggestion

```
    
```

## Community Projects

> - [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence/tree/main/users/) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 

Reviewer: ```suggestion
- [vast2nix](https://github.com/GTrunSec/data-science-threat-intelligence) -  A Nix-based DevOps framework for defining declarative and reproducible environments and jobs. 
```

Author: Sorry about that. The `data-science-threat-intelligence` is another story.
